frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/BrowseNavSection.tsx

```diff
@@ -24,6 +29,9 @@ export const BrowseNavSection = ({
     "expand-browse-in-nav",
   );
 
+  const entityTypes = useSelector(state => getEmbedOptions(state).entity_types);
```

Comment:
on basis, the condition structure should be very similar to https://github.com/metabase/metabase/pull/55331. With some different conditions. See the [conditions for this PR here.](https://www.notion.so/metabase/Tech-Let-embedders-hide-tables-1ae69354c9018057bc1ddcf987443164?pvs=4#1ae69354c9018082bb10fb315b1d209b)